src/10_functions.py

```diff
@@ -1,7 +1,11 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
-
+def is_even(number):
```

Comment:
They are looking for 2 separate functions here. 1 that returns true if a passed number is even. The other function should handle all the printing based on that.